src/template.js

```diff
@@ -13,7 +13,6 @@ const repDeclarationUnder12Deceased = require('./lib/resource/sections/rep-decla
 const repDeclaration12AndOver = require('./lib/resource/sections/rep-declaration-12-and-over.js');
 const repDeclaration12AndOverDeceased = require('./lib/resource/sections/rep-declaration-12-and-over-deceased');
 const applicantConfirmationMethod = require('./lib/resource/sections/applicant-confirmation-method.js');
-const applicantBritishCitizenOrEuNational = require('./lib/resource/sections/applicant-british-citizen-or-eu-national.js');
```

Comment:
Is this now a redundant schema, @BarryPiccinni ??

Reply:
Yep, there were a few schemas being imported that we don't use anymore.